plugins/woocommerce-admin/client/marketplace/components/category-selector/category-selector.tsx

```diff
@@ -0,0 +1,126 @@
+/**
+ * External dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import CategoryLink from './category-link';
+import './category-selector.scss';
+import CategoryDropdown from './category-dropdown';
+import { Spinner } from '@wordpress/components';
```

Comment:
This is being flagged by ESLint for being under "Internal dependencies" instead of "External dependencies"